### ISSUE TITLE
Please make it possible to reenable original v1.0 invasion timers

### ISSUE THREAD

**Issue by maprezentalok** [alexbatalov/fallout1-ce#190]
Hi,

is it possible to reenable the time limit of super mutant invasion of towns through some ini or configuration file?
I find your project the best way to play classic fallouts, but this option would be really neat.

Thanks for your fantastic work, keep it up! :)

"Fallout originally had a more dynamic wasteland, with the super mutants expanding and invading settlements across the wasteland as time passed by. The longer the player took to complete the game, the more settlements would be invaded, with all characters removed, with only super mutant invaders present. A remnant of this mechanic can be seen with Necropolis, where entering after 110 days pass triggers the invasion: All ghouls are killed off by script, with super mutant invaders added to surface maps, along with a ghoul refugee who recounts the invasion. The time limits are still included in Vault13.gam and are as follows:

- Followers of the Apocalypse would be invaded 90 days after the start of the game (March 5, 2162).
- Necropolis falls after 110 days (March 25, 2162).
- The Hub falls after 140 days (April 24, 2162).
- Lost Hills is invaded after 170 days (May 24, 2162).
- Junktown is invaded after 200 days (June 23, 2162).
- Shady Sands falls after 230 days (July 23, 2162).
- Vault 13 finally falls and the game ends in a fail state after 260 days pass (August 22, 2162). This was modified to 500 days for release (corresponding to April 19, 2163)."

**Comment by dje4321** [comment]
To my knowledge, those invasion timers were not vanilla but were more of a planned feature. Only Necropolis and the Vault still have their invasion timers implemented with the vaults being dependant on the water merchants. 

Regardless, not something that you implement at an engine level as you would need to rework the game script files.

**Comment by maprezentalok** [comment]
Yes now I remember, vanilla v1.0 only had the Necropolis (110 days) and Vault (500 or 400 days with water merchants) timer, and the latter was patched out. Alright, I will try to isolate the script within Fixt, thanks guys!

**Comment by emxd** [comment]
> You can probably just use the TeamX Restoration mod to get those, it's not binary dependent. At least it's worth a try.

I was playing around with diffferent Fallout versions and I thought of trying this. And it seems to work ok, I just applied teamx 1.2.1 + 1.3.5 + teamx restoration mod and I got the invasions and the 500 days limit working. I did forgot though that the TeamX Restoration Mod was distributed with a patched TeamX 1.2 executable (thought that it was juts SSL scripts). Not sure what precisely they patched there.

But the invasions/timers seem to be entirely SSL based so it worked on f1-ce fine.
